src/renderer/d3d.rs

```diff
@@ -120,7 +126,7 @@ impl D3DSkiaRenderer {
         };
 
         #[cfg(not(feature = "d3d_debug"))]
-        let dxgi_factory: IDXGIFactory2 =
+        let dxgi_factory: IDXGIFactory4 =
```

Comment:
I don't think we can change this. See https://github.com/neovide/neovide/pull/2559

Reply:
sure, i'll rollback to `IDXGIFactory2`